src/main/scala/lenthall/config/ScalaConfig.scala

```diff
@@ -8,6 +10,7 @@ object ScalaConfig {
   implicit class EnhancedScalaConfig(val config: Config) extends AnyVal {
     def getConfigOption(key: String): Option[Config] = getOption(key, config.getConfig)
     def getStringOption(key: String): Option[String] = getOption(key, config.getString)
+    def getPathOption(key: String): Option[Path] = getStringOption(key) map { Paths.get(_) }
```

Comment:
I would prefer this to be `getFSPathOption` since `Path` is already a config file concept and I got confused here for an entire second